core/vm/contracts.go

```diff
@@ -142,6 +142,22 @@ func ActivePrecompiles(rules params.Rules) []common.Address {
 	}
 }
 
+func GetPrecompile(rules params.Rules, addr common.Address) (PrecompiledContract, bool) {
```

Comment:
docstring

Reply:
Also, maybe name it `ActivePrecompiles`

Reply:
Oh yeah sry forgot.
`ActivePrecompiles` is taken (see a few lines above). What do you think of `GetActivePrecompile`?

Reply:
Wait.. Why don't you just use `ActivePrecompiles` instead? You need to iterate a slice to check if a given address is in there...  but it's a fairly short slice

Reply:
For some reason I thought `ActivePrecompiles` returns the precompile objects and not the addresses...Re-wrote it and I like it better now too